tests/src/whisk/core/entity/test/ConformanceTests.scala

```diff
@@ -114,7 +114,7 @@ class ConformanceTests extends FlatSpec
 
         checkDatabaseFields(datastore, "whisks/all", isTrigger, Set("limits", "parameters"))
 
-        checkDatabaseFields(datastore, "whisks/all", isRule, Set("status", "trigger", "action"))
+        checkDatabaseFields(datastore, "whisks/all", isRule, Set("trigger", "action"))
```

Comment:
@markusthoemmes this needed to be done for new activator - missed it.
@psuter FYI.